README.md

```diff
@@ -1 +1,16 @@
-Destination folder for xcconfig files
+# Epi PSK Node Server
+
+PSKNodeServer project for ePI app - this project contains API for scanning barcodes
+
+## How to install and setup
+
+### Preconditions
+
+- Xcode 12.5 (Swift 5.4)
+- Carthage
+  - `brew install carthage`
+
+### Setup
+
+- Clone this repository
+- Run `Carthage bootstrap` in the root of PSKNodeServer to install Carthage dependencies
```

Comment:
`Carthage` has to start with small letter `c`

Reply:
I would specify `in the root of PSKNodeServer folder`. Just to be clear that you mean folder not some server :D